files/bin/buildlibs.sh

```diff
@@ -21,8 +21,7 @@ else
 fi
 
 DEPS_DIR=${DEPS_DIR:-${HOME}/.local/c0}
-
-PKG_LIBS=${1}/lib/${OS}-${NIMARCH}
+PKG_LIBS=${1:-"./files/deps"}/lib/${OS}-${NIMARCH}
```

Comment:
I'm not a huge fan of the default string I have provided here but not sure what would be better. V open to suggestions